Resources/Lang/en_US.json

```diff
@@ -3005,12 +3005,12 @@
   "Councillor_MurderKillTitle": "WICKED COURT ",
   "CouncillorMakeEvilJudgeClear": "Show Trial as Councillor Murder",
   "Councillor_CannotMurderImpTeam": "Sorry, you can not murder your teammate.",
-  "Councillor_SuidiceForMurderImps": "You died because you are trying to murder your team members.",
+  "Councillor_SuicideForMurderImps": "You died because you are trying to murder your team members.",
```

Comment:
When the translation key changes, the name in the code where it is used also needs to be changed
![Screenshot_20240723_082946](https://github.com/user-attachments/assets/fbc914e4-0113-47e2-9816-a335d2423bca)

Reply:
Done